server/views/temporary-accommodation/bedspace-search/results.njk

```diff
@@ -135,6 +135,14 @@
                                     </div>
                                 </div>
 
+                                <div class="govuk-grid-row">
+                                    <div class="govuk-grid-column-full">
+                                        <h3 class="govuk-heading-s">Property notes</h3>
+                                        <div data-cy-premises-notes>
+                                            {{ result.premises.notes }}
+                                        </div>
```

Comment:
I think it might be better to place the content inside `p` rather than div. Speaking of which could you tryout notes that have paragraphs in them and some other formatting?

```suggestion
                                        <p data-cy-premises-notes>
                                            {{ result.premises.notes }}
                                        </p>
```

Reply:
Yea i wasn't sure if the notes may include <p> so went safe. I'll look at other formatting on notes.